infra/app/checkout.parameters.json

```
@@ -0,0 +1,24 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "name": {
+      "value": "${SERVICE_CHECKOUT_NAME}"
```

```suggestion
      "value": "${SERVICE_PUBLISHER_NAME}"
```

Whole file needs this change.  and upstream